### DISCUSSION TITLE
Error since a week ... " Keine Geräte oder Entitäten Fehler beim Einrichten: Überprüfe die Protokolle"

### DISCUSSION THREAD

**Discussion by higgy-de** [Hyundai-Kia-Connect/kia_uvo#900] [General]
Hello,
since a week my Homeassistant dont connect to my EV6.

Dont know how to search the problem ... who can help me ?

![Zwischenablage05](https://github.com/Hyundai-Kia-Connect/kia_uvo/assets/158240781/454986d7-3032-4f4e-b095-9bbc7268ee9e)

**Comment by cdnninja** [comment]
Open an issue and follow the template.

**Reply by higgy-de** [reply]
Hello ,
thanks for answer ... you mean here in github ? sorry ...i am new to this.

**Reply by cdnninja** [reply]
Yes.  Debug logs being key.  Native app working?
